webapp/lib/controller.dart

```diff
@@ -690,16 +778,45 @@ void setMessageTag(model.Tag tag, model.Message message, model.Conversation conv
   }
 }
 
-Set<model.Conversation> filterConversationsByTags(Set<model.Conversation> conversations, List<model.Tag> filterTags, DateTime afterDateFilter) {
-  if (filterTags.isEmpty && afterDateFilter == null) return conversations;
+Set<model.Conversation> filterConversationsByTags(Set<model.Conversation> conversations, ConversationFilter conversationFilter) {
+  if (conversationFilter.includeTags.isEmpty &&
+      conversationFilter.excludeTags.isEmpty &&
+      conversationFilter.includeAfterDateFilter == null &&
+      conversationFilter.excludeAfterDateFilter == null) {
```

Comment:
Perhaps `if (conversationFilter.isEmpty)` or something like that? 
I was looking for ways to make code easier to read by adding utility methods to the `ConversationFIlter` class.

Reply:
Great idea, thanks!